docs/guides/akamai/get-started/observability-with-datastream-and-trafficpeak/index.md

```diff
@@ -0,0 +1,93 @@
+
+## TrafficPeak On Akamai Cloud
+
+### What Is TrafficPeak?
+
+TrafficPeak is a fully managed observability solution that works with DataStream log streaming and Akamai Cloud Computing. TrafficPeak is managed and hosted by Hydrolix, and uses Linode Compute Instances alongside Linode Object Storage for data processing and storage. With TrafficPeak, customers are provided with access to a Grafana interface with preconfigured, customizable dashboards for data visualization and monitoring.
+
+### Who Is TrafficPeak For?
+
+TrafficPeak is for Akamai customers that need an all-in-one, cost-effective, turnkey observability solution for large, petabyte-scale volumes of data.
+
+For more detailed information on features and pricing, see: [Observability on Akamai cloud computing: TrafficPeak](https://hydrolix.io/partner-program/trafficpeak/)
+
+## Overcoming Challenges
+
+### Cost Reduction & Visibility
+
+*Reduce observability costs by up to 75% while standardizing data visualization.*
+
+Where other observability solutions are built for application-specific observability, TrafficPeak is built to achieve CDN-level observability supporting extremely large amounts of data. TrafficPeak is designed to work with DataStream to ingest logs from the edge and uses Compute Instances with Object Storage to achieve an efficient, decoupled, and stateless architecture. By keeping traffic and CDN logs on the same platform, TrafficPeak can reduce your observability costs by up to 75%.
+
+With large amounts of data, the need for standardizing data reporting becomes even more important. TrafficPeak’s Grafana dashboard supports as many users you may need, providing a centralized data source. This helps streamline organizational operations and allows developers that work with the data the ability to all get it from the same place.
+
+### Large Data Volumes & Data Retention
+
+*Address data volume and retention challenges with an observability solution on the same platform as your edge delivery.*
+
+There are numerous cost and infrastructure-based challenges that are associated with large volumes of data, including: data acquisition, data storage and retention, data searching and efficiency, and more. Since TrafficPeak is built for Akamai, you can address these challenges by coupling your edge and observability solutions on a single platform.
+
+With TrafficPeak, logs are sent directly from Akamai edge to Linode Compute using DataStream, eliminating the need for data traffic to travel outside the Akamai platform. TrafficPeak separates log processing from log storage by utilizing Compute Instances and Object Storage in tandem, uses up to 25x data compression, and offers searchable, accessible hot data retention for 15+ months.
```

Comment:
We mention that log processing and log storage are separated. Is this what is meant by a 'decoupled' architecture as described on line 41? If so, I think it could be helpful to call that out here

Reply:
going to leave as is just to avoid any technical mis-speaking